engine/src/main/java/org/hawkular/alerts/engine/impl/ispn/IspnAlertsServiceImpl.java

```diff
@@ -182,7 +184,8 @@ public Alert getAlert(String tenantId, String alertId, boolean thin) throws Exce
             log.debugf("getAlerts criteria: %s", criteria.toString());
         }
 
-        StringBuilder query = new StringBuilder("from org.hawkular.alerts.api.model.event.Alert where ");
+        StringBuilder query = new StringBuilder("from org.hawkular.alerts.engine.impl.ispn.model.IspnEvent where ");
+        query.append("eventType = 'ALERT' and ");
```

Comment:
I'm not sure how Lucene does things and whether the order of criteria matters.  But to keep our approach of most restrictive criteria first we'd want to move this condition to the end, as the field has only 2 possible values.